include/deal.II/lac/sparse_direct.h

```diff
@@ -474,13 +475,11 @@ class SparseDirectMUMPS : public EnableObserverPointer
         : blr_ucfs(blr_ucfs)
         , lowrank_threshold(lowrank_threshold)
       {}
-
```

Comment:
Can you undo the whitespace changes like this one and the few above and below? (Of course, if there is a similar pattern elsewhere, feel free to keep it. But at least below line 519 I see one empty line between member variables.)